packages/app-store/googlecalendar/api/callback.ts

```diff
@@ -36,10 +36,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
   if (code) {
     const token = await oAuth2Client.getToken(code);
-
     key = token.res?.data;
-
-    if (!key) res.status(401).json({ message: "Permissions not granted" });
```

Comment:
We have already covered this scenario afterwards in the workflow. This result in the API was throwing a bare bones JSON when clicking "Cancel" in Google Calendar permission flow, not a very good UX.